EIPS/eip-5218.md

```diff
@@ -0,0 +1,252 @@
+---
+eip: 5218
+title: NFT Rights Management Standard
+description: An interface for creating copyright licenses that transfer with an NFT.
+status: Draft
+type: Standards Track
+category: ERC
+author: James Grimmelmann <james.grimmelmann@cornell.edu>, Yan Ji <jyamy42@gmail.com>, Tyler Kell <sk3259@cornell.edu>
+discussions-to: https://github.com/ethereum/eips/issues/5218
```

Comment:
Must be a thread on ethereum magicians

Reply:
Thank you for editing the urls! I've created a thread on ethereum magicians and updated the link.